charts/images.yaml

```diff
@@ -4,10 +4,6 @@ images:
   repository: eu.gcr.io/gardener-project/gardener/terraformer-aws
   tag: "v2.17.0"
 
-- name: cloud-controller-manager
-  sourceRepository: github.com/kubernetes/kubernetes
-  repository: k8s.gcr.io/hyperkube
-  targetVersion: "< 1.17"
```

Comment:
Can we delay this cleanup for a separate PR ? Because I assume we need to do much more changes to cleanup handlings for K8s < 1.17 (incl. updating the supported versions in the README.md).